src/app/locator/qgsinbuiltlocatorfilters.h

```diff
@@ -151,6 +151,32 @@ class QgsBookmarkLocatorFilter : public QgsLocatorFilter
 
 };
 
+class QgsSettingsLocatorFilter : public QgsLocatorFilter
+{
+    Q_OBJECT
+
+  public:
+
+    QgsSettingsLocatorFilter( QObject *parent = nullptr );
+    QgsSettingsLocatorFilter *clone() const override;
+    QString name() const override { return QStringLiteral( "optionpages" ); }
+    QString displayName() const override { return tr( "Settings" ); }
+    Priority priority() const override { return Highest; }
+    QString prefix() const override { return QStringLiteral( "s" ); }
```

Comment:
I reluctant to reserve the "s" keyword for this use. I think we need to be very conservative here about using the single letter prefixes, because we've got a very limited pool to use and accordingly they should be kept for "heavily used" searches. I'd ever leave no prefix by default or go with something more explicit like "set".

Reply:
Will change to "set".

May I suggest that we find a way to internationalize these keywords? _Settings_ in danish is _Indstillinger_ and neither _s_ nor _set_ is descriptive.

I know that we somehow _need_ to ensure that keywords are unique even though they've been authored in transifix. Maybe we can implement a fallback mechanism based on 1: displayname 2: the english keyword.

Reply:
Hmm... tricky! If we allow them to be translated then we run into very high chance of clashing shortcuts. I think I'd rather have some way users can configure them themselves to avoid this situation (this could also allow users to manually create single character searches for plugin based filters)